### PR TITLE
Add probe latency and incarnation number metrics to SWIM Membership

### LINKED ISSUE #6003: Add metrics to SwimMembershipProtocol
**Description**

To help tune the config parameters of Swim protocol, it would be useful to add some metrics. 
Examples:
- Probe latency - RTT for a probe request
- Gossip latency - how long until a metadata change on a node is propagated to another node.

Related to https://github.com/zeebe-io/zeebe/issues/4827#issuecomment-740624583 https://github.com/zeebe-io/zeebe/issues/4827#issuecomment-740768778

### PR DESCRIPTION
**Pull request by rodrigo-lourenco-lopes**
## Description

This PR adds 2 SWIM protocol metrics and their panels respectively in the Grafana Dashboards.
These metrics are: 

- Execution time of the SWIM protocol probe (time taken between sending the probe and getting the acknolege)
- Swim protocol Members' incarnation number. This metric is used as a way to enable us to see the incarnation number being broadcasted through the network and therefore derive the time it takes for the gossip of new information to spread. This metric has a minimum granularity of the time Prometheus polls these metrics from the members (the current one is 15s).

image

## Related issues

closes #6003 

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [x] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Operate](https://github.com/camunda/operate/issues)
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -15635,6 +15550,228 @@
...
+          "pluginVersion": "9.3.6",
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "${DS_PROMETHEUS}"
+              },
+              "editorMode": "code",
+              "expr": "sum(zeebe_smp_members_incarnation_number) by (pod, address)",
+              "legendFormat": "{{pod}} => {{address}} ",
```
Comment (deepthidevaki) [inline monitor/grafana/zeebe.json:15764]: :x: Use a meaningful legend. Since you are adding "member" as the label, you could just use it.

**Review by deepthidevaki — CHANGES_REQUESTED** [review]
Thanks Rodrigo. Please see my comments. We can also discuss it f2f. 

Besides that

- Please update the PR title to a more meaningful one
- Please link the issue to the PR description in closes.

File: monitor/grafana/zeebe.json [hunk 2]
```diff
@@ -1,83 +1,4 @@
 {
-  "__inputs": [
```
Comment (deepthidevaki) [inline monitor/grafana/zeebe.json:2]: :question: Could you please verify if removing this is ok?

**Review by deepthidevaki — APPROVED** [review]
:rocket:

File: monitor/grafana/zeebe.json (hunk 2 above)
Comment (rodrigo-lourenco-lopes) [reply monitor/grafana/zeebe.json:2]: I imported the file to a new grafana dashboard and everything seems to be working 👍

**Pull request merged**